src/main/java/gov/cms/madie/resources/PackageController.java

```diff
@@ -51,10 +54,13 @@ public byte[] getMeasurePackage(
       },
       consumes = {MediaType.APPLICATION_JSON_VALUE})
   public String getMeasureSimpleXml(
-      @RequestBody @Validated(Measure.ValidationSequence.class) Measure measure)
+      @RequestBody @Validated(Measure.ValidationSequence.class) Measure measure,
+      @RequestHeader("Authorization") String accessToken)
       throws JAXBException {
```

Comment:
I am unable to get a full picture of who is the client of "/simple-xml" resource, but do you guys think throwing an exception from Controller is ok ? I just feel like we shouldn't be in a situation where something is failing and we have no idea, because we are not even logging it

Reply:
We can ignore it if not important for this release

Reply:
That's a good point - we should have that caught by the controller advice. I don't think any other service or the UI would consume the SimpleXML directly now (or maybe ever) but it is a good debugging endpoint since it's the input to the HQMF. As the HQMF/SimpleXML generation is still somewhat of a WIP, we can tackle this during the sprint that started today

Reply:
yeah, right this is kind of test end point for us for now. but I've addressed it now. Thank Rohit!